wiki/Beatmap_search/en.md

```diff
@@ -86,6 +90,8 @@ Beatmap metadata fields can be compared against specific values for more granula
 | `length` | [Play time](/wiki/Beatmap/Play_time) in seconds |
 | `key`, `keys` | Number of keys (osu!mania and converted beatmaps only) |
 | `status` | Beatmap status. Value can be `ranked`, `approved`, `pending`, `notsubmitted`, `unknown`, or `loved`, or `r`/`a`/`p`/`n`/`u`/`l` for short. |
+| `played`, `lastplayed` | Time since last played. Accepts a format of `#y#M#d#h#m#s`, for years, months, days, hours, minutes, and seconds respectively. For example,`2d5s` means "2 days and 5 seconds". |
```

Comment:
available in the next lazer update

Reply:
does https://github.com/ppy/osu-wiki/pull/10994/commits/8f37910d6794021ac1e83a176763abe3bae0a0e6 work?